code/datums/records/record.dm

```diff
@@ -0,0 +1,615 @@
+	var/lock_ref
+	/// Names of major disabilities
+	var/major_disabilities
+	/// Fancy description of major disabilities
+	var/major_disabilities_desc
+	/// List of medical notes
+	var/list/medical_notes = list()
+	/// Names of minor disabilities
+	var/minor_disabilities
+	/// Fancy description of minor disabilities
+	var/minor_disabilities_desc
+	/// Physical status of this person in medical records.
+	var/physical_status
+	/// Mental status of this person in medical records.
+	var/mental_status
+	/// Positive and neutral quirk strings
+	var/quirk_notes
+	/// Security note
+	var/security_note
+	/// Current arrest status
+	var/wanted_status = WANTED_NONE
+
+	///Photo used for records, which we store here so we don't have to constantly make more of.
+	var/list/obj/item/photo/record_photos
+
+/datum/record/crew/New(
+	age = 18,
+	blood_type = "?",
+	character_appearance,
+	dna_string = "Unknown",
+	fingerprint = "?????",
+	gender = "Other",
+	initial_rank = "Unassigned",
+	name = "Unknown",
+	rank = "Unassigned",
+	hud = "None",
+	active_department = NONE,
+	species = "Human",
+	/// Crew specific
+	lock_ref,
+	medical_notes,
+	major_disabilities = "None",
+	major_disabilities_desc = "No disabilities have been diagnosed at the moment.",
+	minor_disabilities = "None",
+	minor_disabilities_desc = "No disabilities have been diagnosed at the moment.",
+	physical_status = PHYSICAL_ACTIVE,
+	mental_status = MENTAL_STABLE,
+	quirk_notes,
+	security_note,
+	wanted_status = WANTED_NONE,
+)
+	. = ..()
+	src.lock_ref = lock_ref
+	src.major_disabilities = major_disabilities
+	src.major_disabilities_desc = major_disabilities_desc
+	src.minor_disabilities = minor_disabilities
+	src.minor_disabilities_desc = minor_disabilities_desc
+	src.physical_status = physical_status
+	src.mental_status = mental_status
+	src.quirk_notes = quirk_notes
+
+	GLOB.manifest.general += src
+
+/datum/record/crew/Destroy()
+	GLOB.manifest.general -= src
+	QDEL_LAZYLIST(record_photos)
+	return ..()
+
+
+/datum/record/crew/proc/get_info_list()
+	var/list/list_of_medical_notes
+	for(var/datum/medical_note/medical_note in medical_notes)
+		list_of_medical_notes += medical_note.get_info_list()
+
+	return list(
+		age = src.age,
+		blood_type = src.blood_type,
+		record_ref = FAST_REF(src),
+		dna = src.dna_string,
+		gender = src.gender,
+		major_disabilities = src.major_disabilities_desc,
+		minor_disabilities = src.minor_disabilities_desc,
+		physical_status = src.physical_status,
+		mental_status = src.mental_status,
+		name = src.name,
+		quirk_notes = src.quirk_notes,
+		rank = src.rank,
+		species = src.species,
+		list_of_medical_notes
+		)
+
+
+/datum/record/crew/proc/add_medical_note(list/params)
+	if(!params["content"])
+		return FALSE
+	var/content = STRIP_HTML_FULL(params["content"], MAX_MESSAGE_LEN)
+
+	var/datum/medical_note/new_note = new(usr.name, content)
+	while(length(medical_notes) > 2)
+		medical_notes.Cut(1, 2)
+
+	medical_notes += new_note
+	return TRUE
```

Comment:
```suggestion
/datum/record/crew/proc/add_medical_note(content_to_add, author_name = "Anonymous")
	if(!content_to_add)
		return FALSE
	var/content = STRIP_HTML_FULL(content_to_add, MAX_MESSAGE_LEN)

	var/datum/medical_note/new_note = new(author_name, content_to_add)
	while(length(medical_notes) > 2)
		medical_notes.Cut(1, 2)

	medical_notes += new_note
	return TRUE
```
Near perfect except for a few points:
* Instead of sending param list directly, the proc should take what it's needed
* Do not use `usr` directly (no `usr.name`). The proc should have the author name directly.

You should do other param-calling procs too. With the change request, it becomes easier to manipulate record data outside of computer (i.e. Admin proc call)